packages/okta-react/test/e2e/harness/src/Home.js

```diff
@@ -19,12 +19,15 @@ export default withAuth(class Home extends Component {
     super(props);
 
     this.state = {
-      authenticated: null
+      authenticated: null,
+      renewMessage: '',
     };
 
     this.checkAuthentication = this.checkAuthentication.bind(this);
     this.login = this.login.bind(this);
     this.logout = this.logout.bind(this);
+    this.renew_idToken = this.renewToken.bind(this, 'idToken');
```

Comment:
Why the underscores?  Mixing snake_case and camelCase leads to platypuses (platypi?)